exercises/static/exercises/follow_line/web-template/assets/controller.js

```diff
@@ -1,35 +1,33 @@
 // Main message controller for websockets
-var running = true;
+var running = false;
 
 // Function to execute the student solution
 function start(){
+    // Manager Websocket
+    if (running == false) {
+        resumeSimulation();
+    }
+
     // Code Websocket
     submitCode();
 
     // GUI Websocket
     unpause_lap();
-
-    // Mark the variables
-    running = true;
 }
 
 // Function to stop the student solution
 function stop(){
-    // Code Websocket
-    stopCode();
```

Comment:
In this part, shouldn't we keep the `stopCode()` function?

If we remove it, then on clicking Stop, the simulation will pause, however the student code will keep on running.